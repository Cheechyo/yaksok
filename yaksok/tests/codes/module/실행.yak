(@계산 1 3 더하기) 보여주기
(@계산 1 3 빼기) 보여주기
(@계산 3 9 곱하기) 보여주기
