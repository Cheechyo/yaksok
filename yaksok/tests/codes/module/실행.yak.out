4
4
-2
-2
27
27
